uh-groupings-api-overrides.skeleton.properties

```diff
@@ -1,27 +1,15 @@
-# Create a directory in your home directory called ".<username for your computer>-conf".
+# Create a directory in your home directory called ".<username for your coputer>-conf".
```

Comment:
Spelling error at "coputer", should be "computer"

Reply:
done, thank you